code/planet/htmltmpl.py

```diff
@@ -1033,14 +1027,14 @@ def read(self, filename):
             try:
                 f = open(filename, "r")
                 data = f.read()
-            except IOError, (errno, errstr):
-                raise TemplateError, "IO error while reading template '%s': "\
-                                     "(%d) %s" % (filename, errno, errstr)
+            except IOError as e:
+                raise TemplateError("IO error while reading template '%s': " \
+                                    "(%d) %s" % (filename, e.errno, e.errstr))
```

Comment:
Automated conversion has failed ya a bit here. If you have a misconfiguration, with a missing template file, you'll hit this and get: 

```
  File "/Users/offby1/projects/planet-python/code/planet/htmltmpl.py", line 981, in compile
    tokens = self.parse(self.read(file))
  File "/Users/offby1/projects/planet-python/code/planet/htmltmpl.py", line 1032, in read
    "(%d) %s" % (filename, e.errno, e.errstr))
AttributeError: 'FileNotFoundError' object has no attribute 'errstr'
```